tool/tsh/common/access_request.go

```diff
@@ -219,6 +219,9 @@ func printRequest(cf *CLIConf, req types.AccessRequest) error {
 		// Display the expiry time in the local timezone. UTC is confusing.
 		table.AddRow([]string{"Access Expires:", req.GetAccessExpiry().Local().Format(time.DateTime)})
 	}
+	if req.GetAssumeTime() != nil {
+		table.AddRow([]string{"Assume Time (UTC):", req.GetAssumeTime().UTC().Format(time.RFC822)})
```

Comment:
Why use a different time format here?

Reply:
This was the timezone used for other tables with times in the output e.g. [here](https://github.com/gravitational/teleport/blob/a57c65549b78c14b1e835cd163b3b5a282a79703/tool/tctl/common/access_request_command.go#L452) and [here](https://github.com/gravitational/teleport/blob/a57c65549b78c14b1e835cd163b3b5a282a79703/tool/tctl/common/alert_command.go#L220)
I wanted to keep the format consistent.